dill/__diff.py

```diff
@@ -57,7 +63,7 @@ def get_seq(obj, cache={str: False, frozenset: False, list: True, set: True,
     """
     Gets all the items in a sequence or return None
     """
-    o_type = type(obj)
+    o_type = type(obj) if type(obj) is not InstanceType else obj.__class__
```

Comment:
```suggestion
    o_type = obj.__class__
```
Changing it to just `obj.__class__` should work?

Reply:
I'm afraid this will fail on old-style class objects (`<type 'classobj'>`), which have no `__class__` attribute.

Reply:
`getattr(obj, "__class__", type(obj))` then? It would be nice to avoid the double call as diff is slow enough as it is...

Reply:
Still it does double work: tries to get an attribute and also calls `type` regardless of the result. I guess the best we can do here performance-wise is
```
try:
    o_type = obj.__class__
except AttributeError:
    o_type = type(obj)
```

Reply:
I guess. Since we are supporting so many python versions I suppose there is only so much we can do.